frontend/packages/employee-mobile-frontend/src/api/attendances.ts

```diff
@@ -55,6 +57,25 @@ interface Absence {
   id: string
 }
 
+interface DailyNote {
+  id: string
+  childId: string | null
+  groupId: string | null
+  date: LocalDate
+  note: string | null
+  feedingNote: DaycareDailyNoteLevelInfo | null
+  sleepingNote: DaycareDailyNoteLevelInfo | null
+  sleepingHours: number | null
+  reminders: DaycareDailyNoteReminder[]
+  reminderNote: string | null
+  modifiedAt: Date | null
+  modifiedBy: string
+}
+
+type DaycareDailyNoteLevelInfo = 'GOOD' | 'MEDIUM' | 'NONE'
+
+type DaycareDailyNoteReminder = 'DIAPERS' | 'CLOTHES' | 'LAUNDRY'
```

Comment:
these are included here https://github.com/espoon-voltti/evaka/pull/549 in a different place

Reply:
employee desktop frontend vs employee mobile frontend